readme.md

```diff
@@ -230,6 +261,5 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+I am kind of unsure of how to convert AR to mongoose. 
```

Comment:
Here's how I'd do it in mongoose:

```js
Instructor.findOne({name: "Andy"), function(err, docs){
  docs.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
  docs.save();
});
```

In the future, even if you're not sure, give it a try... even if you get it wrong, that helps us understand what your not quite getting, so I can give more targeted feedback.